### PR TITLE
fix latinise1.1

### PR DESCRIPTION
**Pull request by itkg-alavieille**

### CODE REVIEW

**Comment by Thiblef** [comment]
:+1:

File: BackofficeBundle/Resources/public/coffee/extendView/generateId.coffee [hunk 1]
```diff
@@ -17,7 +17,7 @@ extendView['generateId'] = {
   generateId: ->
     if @checkInputId()
       contentTypeId = @recupInput($(".generate-id-source"))
```
Comment (itkg-canne) [inline BackofficeBundle/Resources/public/coffee/extendView/generateId.coffee]: Is the recupInput function really needed ? At least, the name should be updated to use english words.

**Comment by itkg-alavieille** [comment]
@itkg-canne

**Comment by itkg-canne** [comment]
:+1:

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**